vscode/microsoft-kiota/src/extension.ts

```diff
@@ -195,22 +195,33 @@ export async function activate(
         }
         if (result && getLogEntriesForLevel(result, LogLevel.critical, LogLevel.error).length === 0) {
           // Save state before opening the new window
-          void context.workspaceState.update('generatedOutput', {
+          const outputState = {
             outputPath,
             config,
             clientClassName: config.clientClassName || config.pluginName
-          } as GeneratedOutputState);
-          if (!vscode.workspace.workspaceFolders || vscode.workspace.workspaceFolders.length === 0) {
-            await vscode.commands.executeCommand('vscode.openFolder', vscode.Uri.file(config.workingDirectory ?? getWorkspaceJsonDirectory()), true);
+          };
+          void context.workspaceState.update('generatedOutput', outputState as GeneratedOutputState);
+
+          const pathOfSpec = path.join(outputPath, `${outputState.clientClassName?.toLowerCase()}-openapi.yml`);
+          const pathPluginManifest = path.join(outputPath, `${outputState.clientClassName?.toLowerCase()}-apiplugin.json`);
+          if (deepLinkParams.source && deepLinkParams.source === 'ttk') {
+            try {
+              await vscode.commands.executeCommand(
+                'fx-extension.createprojectfromkiota',
```

Comment:
I'm guessing this is still a placeholder?

Reply:
This IS the command we should call from ttk

Reply:
I'd have expected their extension ID to start with ms- so it can be identified as a Microsoft extension.
In fact, the marketplace [lists](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) `ms-teams-vscode-extension`
Have you tested the handover? Not saying it's impossible, just they might not be following conventions here and we should double check.